### DISCUSSION TITLE
import { Form } from 'ant-design-vue'  生产环境下useInjectFormItemContext undefined

### DISCUSSION THREAD

**Discussion by whateveryoudo** [vueComponent/ant-design-vue#6613] [Q&A]
使用useInjectFormItemContext，本地正常，build后发现，并未走index.jsx进行useInjectFormItemContext挂载（PS:是否和包含的组件使用defineAsyncComponent 有关）
<img width="588" alt="微信截图_20230529112238" src="https://github.com/vueComponent/ant-design-vue/assets/30063217/a158a682-d671-4247-9af3-4142825c39a3">

**Comment by li-luyi** [comment]
你好，请问你这个问题解决了吗，我也遇到打包后Form对象找不到useInjectFormItemContext方法的问题，本地是正常的
